client/lib/domains/registration/availability-messages.js

```diff
@@ -13,14 +12,122 @@ import { translate } from 'i18n-calypso';
+			message = translate(
+				"{{strong}}%(domain)s{{/strong}} is already connected to someone else's WordPress.com site.",
+				{
+					args: { domain },
+					components: {
+						strong: <strong />,
+					},
+				}
+			);
+			break;
+		case domainAvailability.REGISTERED_SAME_SITE:
+			message = translate( '{{strong}}%(domain)s{{/strong}} is already registered on this site.', {
+				args: { domain },
+				components: {
+					strong: <strong />,
+				},
+			} );
+			break;
+		case domainAvailability.REGISTERED_OTHER_SITE_SAME_USER:
+			message = translate(
+				'{{strong}}%(domain)s{{/strong}} is already registered on your site %(site)s. Do you want to move it to this site? ' +
+					'{{a}}Yes, move it to this site.{{/a}}',
+				{
+					args: { domain, site },
+					components: {
+						strong: <strong />,
+						a: (
+							<a
+								rel="noopener noreferrer"
+								href={ paths.domainManagementTransferToOtherSite( site, domain ) }
+							/>
+						),
+					},
+				}
+			);
+			break;
+		case domainAvailability.MAPPED_SAME_SITE_TRANSFERRABLE:
+			message = translate(
+				'{{strong}}%(domain)s{{/strong}} is already connected to this site, but registered somewhere else. Do you want to move ' +
+					'it from your current domain provider to WordPress.com so you can manage the domain and the site ' +
+					'together? {{a}}Yes, transfer it to WordPress.com.{{/a}}',
+				{
+					args: { domain },
+					components: {
+						strong: <strong />,
+						a: <a rel="noopener noreferrer" href={ paths.domainTransferIn( site, domain ) } />,
+					},
+				}
+			);
+			break;
+		case domainAvailability.MAPPED_SAME_SITE_NOT_TRANSFERRABLE:
+			message = translate( '{{strong}}%(domain)s{{/strong}} is already connected to this site.', {
+				args: { domain },
+				components: {
+					strong: <strong />,
+				},
+			} );
+			break;
+		case domainAvailability.MAPPED_OTHER_SITE_SAME_USER:
+			message = translate(
+				'{{strong}}%(domain)s{{/strong}} is already connected to your site %(site)s. If you want to connect it to this site ' +
+					'instead, we will be happy to help you do that. {{a}}Contact us.{{/a}}',
+				{
+					args: { domain, site },
+					components: {
+						strong: <strong />,
+						a: <a rel="noopener noreferrer" href={ support.CALYPSO_CONTACT } />,
+					},
+				}
+			);
+			break;
+		case domainAvailability.TRANSFER_PENDING_SAME_USER:
+			message = translate(
+				'{{strong}}%(domain)s{{/strong}} is pending transfer. {{a}}Check the transfer status{{/a}} to learn more.',
+				{
+					args: { domain },
+					components: {
+						strong: <strong />,
+						a: (
+							<a
+								rel="noopener noreferrer"
+								href={ paths.domainManagementTransferIn( site, domain ) }
+							/>
+						),
+					},
+				}
+			);
+			break;
+		case domainAvailability.TRANSFER_PENDING:
+			message = translate(
+				"{{strong}}%(domain)s{{/strong}} is pending transfer and can't be connected to WordPress.com right now. " +
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( '{{strong}}%(domain)s{{/strong}} is pending transfer and can't be connected to WordPress.com right now. {{a}}Learn more{{/a}}.' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).